File: resolucoes/TypeScript/lista01/ex009.ts
import prompt from "prompt-sync"

export function main() {
    const p = prompt()

    const reais = Number(p("Digite quanto você tem na carteira:"))
    
    const dolares = reais / 3.45
    
    console.log(`Com R$:${reais} você pode comprar US$:${dolares.toFixed(2)}`)
}

File: resolucoes/Lua/lista01/ex009.lua
print("Digite o valor em reais que voce tem na carteira:")
reais = tonumber(io.read()) -- Lê o valor em reais e converte para um número

dolares = reais / 3.45 -- Calcula o valor em dólares usando a taxa de câmbio

-- Mostra o resultado na tela
print("Com R$" .. reais .. ", voce pode comprar US$" .. string.format("%.2f", dolares))
